llvm.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+################################################################################
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+################################################################################
+#
+# This script will install the llvm toolchain on the different 
+# Debian and Ubuntu versions
+
+set -eux
+
+# read optional command line argument
+LLVM_VERSION=9
+if [ "$#" -eq 1 ]; then
+    LLVM_VERSION=$1
+fi
+
+DISTRO=$(lsb_release -a | grep "Distributor ID" | awk '{print $3}')
+VERSION=$(lsb_release -a | grep Release | awk '{print $2}')
+DIST_VERSION="${DISTRO}_${VERSION}"
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root!" 
+   exit 1
+fi
+
+declare -A LLVM_VERSION_PATTERNS
+LLVM_VERSION_PATTERNS[8]="-8"
```

Comment:
We will have to remember that then.
Maybe it could download a file to extract this info ? (v2 is fine)

Reply:
Do you have this data on versions somewhere in a machine-readable format?
One option would be to update the script on the fly when some of the data changes...